pages/_app.js

```diff
@@ -3,21 +3,41 @@ import '../styles/global.css';
 import { CacheProvider } from '@emotion/react';
 import { CssBaseline, ThemeProvider } from '@mui/material';
 import Head from 'next/head';
+import { useRouter } from 'next/router';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useEffect } from 'react';
+import TagManager from 'react-gtm-module';
 
 import Footer from '../components/layout/Footer.tsx';
 import Header from '../components/layout/Header';
 import theme from '../styles/themes/theme.tsx';
 import createEmotionCache from '../utils/createEmotionCache';
 
 const clientSideEmotionCache = createEmotionCache();
+const gtmId = process.env.NEXT_PUBLIC_GTM_ID;
```

Comment:
Can you add this new variable with an example value in `.env.example` please! And also update everyone in #cop-dev slack with how to get a real version for their local `.env`.

Reply:
Hello Eleni,
I will add the new value in .env.example. I would also send everyone in #cop-dev with the gtmId so they can update their local env file appropriately.